context/AuthContextProvider.test.tsx

```diff
@@ -32,21 +32,29 @@ describe('AuthContextProvider', () => {
     password: 'password1234',
   });
 
+  const setIsSignedIn = jest.fn();
+  const resetUser = jest.fn();
+
   beforeEach(() => {
     jest.clearAllMocks();
   });
 
   test('should show success notification after a successful login', async () => {
-
     mockCreateEmailPasswordSession.mockResolvedValue({});
 
-    await loginAccount({user, router, getUser});
+    await loginAccount({ user, router, getUser });
 
-    expect(mockCreateEmailPasswordSession).toHaveBeenCalledWith(user.email, user.password);
+    expect(mockCreateEmailPasswordSession).toHaveBeenCalledWith(
+      user.email,
+      user.password,
+    );
     expect(getUser).toHaveBeenCalled();
     expect(router.push).toHaveBeenCalledWith('/league/all');
     expect(toast.custom).toHaveBeenCalledWith(
-        <Alert variant={AlertVariants.Success} message="You've successfully logged in!" />,
+      <Alert
+        variant={AlertVariants.Success}
+        message="You've successfully logged in!"
```

Comment:
There seems to be some inconsistency between using double-quotes and singles quotes. For example, here you use double-quotes, but `toHaveBeenCalledWith()` uses single quotes.